src/plugins/data/public/index.ts

```diff
@@ -36,3 +36,16 @@ export * from './types';
 export { IRequestTypesMap, IResponseTypesMap } from './search';
 export * from './search';
 export * from './query';
+export {
+  CONTAINS_SPACES,
```

Comment:
You can rename this to CONTAINS_SPACES_KEY

Reply:
Done